packages/eslint-config-react-app/base.js

```diff
@@ -41,8 +50,10 @@ module.exports = {
   },
 
   rules: {
-    'react/jsx-uses-react': 'warn',
     'react/jsx-uses-vars': 'warn',
-    'react/react-in-jsx-scope': 'error',
+    'react/jsx-uses-react': 'warn',
```

Comment:
This won't be needed now, but if we move it to the below block, it would say that it's an unused import.

Is that what we want here?

Reply:
According to this, it won't hurt if users still import React... but they don't _need_ to do it. So I'm not sure what's the best path here.
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports